helpers/test/test_hopenai.py

```diff
@@ -180,12 +184,12 @@ def _get_dummy_openai_response2() -> dict:
 
 
 # #############################################################################
-# BaseOpenAICacheTest
+# _OpenAICacheTestCase
```

Comment:
Ok although we want to have cohesive PRs and avoid mixing renaming with other stuff